README.md

```diff
@@ -46,8 +46,9 @@ Note: Currently the Dashboard does not have complete coverage of Apache APISIX f
 
 ## Demo
 
+[Online Playground](https://106.55.144.26/)
```

Comment:
```suggestion
[Online Playground](http://106.55.144.26/)
```

Seems this playground only support `http`

Reply:
Good catch!

Reply:
> 接得好！I changed https://106.55.144.26 / to http://106.55.144.26 / and submitted it again, did I receive a new PR?